src/main.c

```diff
@@ -134,8 +136,12 @@ int main(int argc, char **argv) {
   lua_setglobal(L, "EXEFILE");
 
 #ifdef __APPLE__
-  set_macos_bundle_resources(L);
+  lua_pushboolean(L, true);
+  lua_setglobal(L, "MACOS");
```

Comment:
~~If we use MACOS instead of MACOS_RESOURCES shouldn't we modify the start.lua file accordingly ? Please look into the project for all occurrences of MACOS_RESOURCES.~~

My bad, I wrote too quickly. I understand the logic now. MACOS_RESOURCES is set only when we create a bundle and MACOS is set on macOS in any case.